bridgeservice/types/types.go

```diff
@@ -140,6 +140,9 @@ type BridgeResponse struct {
 
 	// Unique hash representing the bridge event, often used as an identifier
 	BridgeHash Hash `json:"bridge_hash" example:"0xabc1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcd"`
+
+	// Address of the transaction sender who initiated the bridge transaction
+	TxnSender Address `json:"txn_sender" example:"0xabc1234567890abcdef1234567890abcdef1234"`
```

Comment:
Adding one more hex digit, so that it is valid Ethereum address format

```suggestion
	TxnSender Address `json:"txn_sender" example:"0xabc1234567890abcdef1234567890abcdef12345"`
```